pkgs/by-name/ff/ffq/package.nix

```
@@ -0,0 +1,41 @@
+{
+  lib,
+  python3,
+  fetchPypi,
+}:
+
+python3.pkgs.buildPythonApplication rec {
+  pname = "ffq";
+  version = "0.3.0";
+  pyproject = true;
+
+  src = fetchPypi {
+    inherit pname version;
```

```suggestion
    pname = "ffq";
    inherit version;
```

`pname` shouldn't be used in fetchers.